opensearch-operator/pkg/helpers/helpers.go

```diff
@@ -16,18 +27,27 @@ func ContainsString(slice []string, s string) bool {
 
 }
 
-// func removeString(slice []string, s string) (result []string) {
-// 	for _, item := range slice {
-// 		if item == s {
-// 			continue
-// 		}
-// 		result = append(result, item)
-// 	}
-// 	return
-// }
+func removeString(slice []string, s string) (result []string) {
```

Comment:
i would create different files of helpers for slices and for fields / reconciler objects